src/pages/Signup/index.tsx

```diff
@@ -26,64 +27,77 @@ function Signup() {
     passwordConfirmErrorMessage,
     onChangePasswordConfirm,
   } = usePassword();
-  const navigate = useNavigate();
+  const [idStatus, setIdStatus] = useState({
+    isValid: false,
+    message: '',
+  });
 
-  const onClickDuplicateCheck = async () => {
-    const response = await axios.post('/api/customers/duplication', { userName: id });
+  const onBlurDuplicateCheck = async () => {
+    try {
+      const {
+        data: { isDuplicate },
+      } = await apiClient.get(`/api/customers/exists?userName=${id}`);
+      const isValid = isDuplicate ? false : true;
+      const message = isDuplicate ? MESSAGES.EXIST_ID : MESSAGES.AVAILABLE_ID;
+
+      setIdStatus({ isValid, message });
```

Comment:
```suggestion
      const message = isDuplicate ? MESSAGES.EXIST_ID : MESSAGES.AVAILABLE_ID;

      setIdStatus({ isValid: !isDuplicate, message });
```
isValid 임시 변수를 만들지 않아도 될 것 같지 않나요? 🙂

Reply:
오오 그러네용 😮